Documentation/Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-notifications.md

```diff
@@ -100,6 +101,12 @@ spec:
     + “none”: message is considered “delivered” if sent to broker
     + “broker”: message is considered “delivered” if acked by broker (default)
 18. `useSSL` (optional) indicates that secure connection will be used for connecting with the broker (“false” by default)
+19. `mechanism` (optional) select which authentication mechanism to use, one of:
+    + “PLAIN”
```

Comment:
please add a note that `PLAIN` is the default mechanism

Reply:
Done!